client/src/main/java/org/apache/rocketmq/client/impl/consumer/DefaultMQPullConsumerImpl.java

```diff
@@ -773,14 +773,6 @@ private void checkConfig() throws MQClientException {
         // check consumerGroup
         Validators.checkGroup(this.defaultMQPullConsumer.getConsumerGroup());
 
-        // consumerGroup
-        if (null == this.defaultMQPullConsumer.getConsumerGroup()) {
-            throw new MQClientException(
-                "consumerGroup is null"
-                    + FAQUrl.suggestTodo(FAQUrl.CLIENT_PARAMETER_CHECK_URL),
-                null);
```

Comment:
IMO exception thrown in `Validators.checkGroup` should keep this more detailed error message.